hyperon_das_atomdb/adapters/server_db.py

```diff
@@ -0,0 +1,277 @@
+import json
+from typing import Any, Dict, List, Optional, Tuple
+
+from requests import Session, exceptions
+
+from hyperon_das_atomdb.exceptions import (
+    ConnectionServerException,
+    NodeDoesNotExistException,
+)
+from hyperon_das_atomdb.i_database import IAtomDB
+from hyperon_das_atomdb.utils.decorators import retry
+
+
+class ServerDB(IAtomDB):
+    """A concrete implementation using servers databases.
+    AwsLambda and OpenFaas"""
+
+    def __repr__(self) -> str:
+        return "<Atom database Server>"  # pragma no cover
+
+    def __init__(
+        self,
+        database_name: str = 'das',
+        ip_address: str = None,
+        port: Optional[str] = None,
+    ) -> None:
+        if not ip_address:
+            raise ConnectionServerException(
+                message='You must send ip_address parameter'
+            )
+        self.database_name = database_name
+        self.ip = ip_address
+        self.port = port if port else '8080'
```

Comment:
Why not using defaults in the parameters of `__init__()`. It would be more explicit to users.

Reply:
Because this port 8080 is not standard for all cases. It is standard for our server on Vultr

Reply:
So having this default hardwired in the middle of the code is even worst. Users will not see it. Having than in the constructor's signature there's a chance that they can see it.